frontend/src/components/documentation/Routes/OAuth/Token.jsx

```diff
@@ -59,6 +59,48 @@ export default class Token extends React.Component {
               Endpoint: <code>https://uclapi.com/oauth/token</code>
             </p>
 
+            <p>Tokens uniquely identify an app that is requesting data from the API.</p>
+
+            <p>Tokens are a long string variable of numbers and letters.
+            e.g. <code>  uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb </code>
+            </p>
+
+            <p>There are two different kinds of tokens you can work with:
+            </p>
+
+            <p>1.    Generic Tokens:
+            These are tokens that are used to request non-personal data. These tokens are used between applications and the API to request any sort of data that the app may need that is not tied to a specific student. For example, <a href="https://uclapi.com/docs#roombookings">UCL API’s Room booking service</a> uses tokens to return information about rooms, when they are booked and which UCL rooms are free.
+            </p>
+
+            <p>2.    OAuth Tokens:
+            This type of token is used when an app requires personal data from users.
+            One of the most common uses of this type of token is when you sign in via UCL on an app. The app will then use a token to request a user’s personal data such as:
+            <ul><li>Department</li>
+            <li> Email </li>
+            <li> Full name </li>
+            <li> Given name </li>
+            <li> UPI </li>
+            <li> If they are a student or not </li>
+            <li> *Student number ~ To get this, you need to tick the relevant scope in the dashboard before a user logs in. ~
+            More on scopes <a href="https://uclapi.com/docs#oauth/meta">here</a>.</li></ul>
+            </p>
+
+            <p>Note that you can also use OAuth Tokens to access all the same data that generic app tokens can access.
+            </p>
+
+            <p>Each token is uniquely generated for each user logging into each app.
+            </p>
+
+            <p>Please note, access to any of this data needs to be approved by the user first.
+            </p>
+
+            <p>To use this type of token for your app, you need include a “Sign in With UCL Button” in your app, such as the one provided below, which links users to the authorisation endpoint with your app’s Client ID (accessible via the dashboard) and a random state number included in the GET parameters.
```

Comment:
The sign in button isn't mandatory only a link to the endpoint. Maybe make clear its optional here?

Reply:
Changed that line - Please let me know what you think now?

Reply:
All good now!